services/resourcemanager/configuration.go

```diff
@@ -22,12 +22,12 @@ func NewConfiguration() *config.Configuration {
 		Debug:         false,
 		Servers: config.ServerConfigurations{
 			{
-				URL:         "https://resource-manager.api.{region}stackit.cloud",
+				URL:         "https://resource-manager.api.stackit.cloud",
```

Comment:
is {region} not needed?

Reply:
For global APIs the variable is not needed, and the generator just removes the variable and sets the default value int the ServerVariable config. For regional APIs it retains the variable (
[here](https://github.com/stackitcloud/stackit-sdk-go/blob/main/services/ske/configuration.go)
for instance)